src/index.html

```diff
@@ -22,6 +22,431 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__link"
+        href="images/logo.png"
```

Comment:
The 'href' attribute in the anchor tag should point to the destination of the link, not the image source. If this is meant to be a logo link, it should typically lead to the site's home page.